ocs_ci/deployment/deployment.py

```diff
@@ -152,51 +136,9 @@ def post_ocp_deploy(self):
         """
         Function does post OCP deployment stuff we need to do.
         """
-        # Workaround for #1777384 - enable container_use_cephfs on RHEL workers
-        # Ticket: RHSTOR-787, see more details in the issue: #1151
-        logger.info("Running WA for ticket: RHSTOR-787")
-        ocp_obj = ocp.OCP()
-        cmd = ['/usr/sbin/setsebool -P container_use_cephfs on']
-        workers = get_typed_worker_nodes(os_id="rhel")
-        for worker in workers:
-            cmd_list = cmd.copy()
-            node = worker.get().get('metadata').get('name')
-            logger.info(
-                f"{node} is a RHEL based worker - applying '{cmd_list}'"
-            )
-            # We saw few times there was an issue to spawn debug RHEL pod.
-            # Let's use retry decorator to make sure our CI is more stable.
-            retry(CommandFailed)(ocp_obj.exec_oc_debug_cmd)(
-                node=node, cmd_list=cmd_list
-            )
-        # end of workaround
-        self.set_registry_to_managed_state()
-        self.add_stage_cert()
-
-    def set_registry_to_managed_state(self):
-        """
-        In order to be able to deploy from stage we need to change
-        image registry config to Managed state.
-        More described in BZs:
-        https://bugzilla.redhat.com/show_bug.cgi?id=1806593
-        https://bugzilla.redhat.com/show_bug.cgi?id=1807471#c3
-        We need to change to managed state as described here:
-        https://github.com/red-hat-storage/ocs-ci/issues/1436
-        So this is not suppose to be deleted as WA case we really need to do
-        this operation for OCS deployment as was originally done here:
-        https://github.com/red-hat-storage/ocs-ci/pull/1437
-        Currently it has to be moved here to enable CA certificate to be
-        properly propagated for the stage deployment as mentioned in BZ.
-        """
-        if(config.ENV_DATA['platform'] not in constants.CLOUD_PLATFORMS):
-            run_cmd(
-                f'oc patch {constants.IMAGE_REGISTRY_CONFIG} --type merge -p '
-                f'\'{{"spec":{{"storage": {{"emptyDir":{{}}}}}}}}\''
-            )
-            run_cmd(
-                f'oc patch {constants.IMAGE_REGISTRY_CONFIG} --type merge -p '
-                f'\'{{"spec":{{"managementState": "Managed"}}}}\''
-            )
+        assert set_selinux_permissions(), "Adding workaround for the issue #1151 failed"
```

Comment:
no need to assert once you remove `return True` from `set_selinux_permissions()`